driver/resthandler.go

```diff
@@ -292,24 +295,23 @@ func (handler RestHandler) newCommandValue(resource model.DeviceResource, readin
 func checkUintValueRange(valueType string, val interface{}) error {
 	switch valueType {
 	case common.ValueTypeUint8:
-		valUint8 := val.(uint8)
-		if valUint8 <= math.MaxUint8 {
+		_, ok := val.(uint8)
+		if ok {
```

Comment:
Should be:
```suggestion
		if !ok {
```
And for the other also.